packages/beeq/src/components/drawer/scss/bq-drawer.scss

```
@@ -0,0 +1,57 @@
+/* -------------------------------------------------------------------------- */
+/*                        Drawer styles                        */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-drawer.variables';
+
+:host {
+  @apply block;
+}
+
+:host(.is-hidden) {
+  @apply hidden;
+}
+
+.bq-drawer {
+  @apply absolute z-20 flex w-[--bq-drawer-width] flex-col overflow-auto bg-bg-primary px-[--bq-drawer--wrapper-X-padding] py-[--bq-drawer--wrapper-Y-padding] shadow-m;
+
+  &.left {
+    @apply inset-y-0 left-0;
+  }
+
+  &.right {
+    @apply inset-y-0 right-0;
+  }
+}
+
+/* Styling for the backdrop */
+.bq-drawer-backdrop {
+  @apply fixed inset-0 z-10 bg-[--bq-drawer--background-backdrop] opacity-60;
+}
+
+/**
+ * Tweak the close bq-buton styles so it remain small wihout extra padding
```

Careful with the typos 😉 

```suggestion
 * Tweak the close bq-button styles so it remain small without extra padding
```